scheduler/test/cook/test/mesos/api.clj

```diff
@@ -1632,3 +1636,34 @@
       (is (nil? (:job/name job-entity)))
       (is (= job-uuid (:uuid job-map-for-api)))
       (is (= "cookjob" (:name job-map-for-api))))))
+
+(deftest test-get-user-usage-no-usage
```

Comment:
Can you add some more tests, for instance where they have usage in some pools but not others?

Reply:
Done